adaptors/backbone/context_adaptor.js

```diff
@@ -32,6 +60,12 @@ var lookupValue = function(view, name) {
   if (this.isModel(view) && view.has(name)) {
     value = view.get(name);
   } else if (view[name]) {
+    if (view.tungstenCollection && blockedCollectionProperties[name]) {
+      return null;
+    }
+    if (view.tungstenModel && blockedModelProperties[name]) {
+      return null;
```

Comment:
Would this be a good place to throw a warning about trying to access a blocked property?